dll/win32/riched20/txtsrv.c

```diff
@@ -527,6 +599,26 @@ const ITextServicesVtbl text_services_stdcall_vtbl =
     NULL,
     NULL,
     NULL,
+#ifdef _MSC_VER
+    ITextServices_TxSendMessage,
```

Comment:
Same question in this file as well.

Reply:
See comment above. Thanks for the suggestion.